ert_storage/database_schema.py

```diff
@@ -93,6 +112,28 @@ class Experiment(Base, MetadataField):
 )
 
 
+class Prior(Base, MetadataField):
+    __tablename__ = "prior"
+
+    pk = sa.Column(sa.Integer, primary_key=True)
+    id = sa.Column(UUID, unique=True, default=uuid4)
+    time_created = sa.Column(sa.DateTime, server_default=func.now())
```

Comment:
Why `time_created` and `time_updated` if these come along with the experiment?

Reply:
Copy-pasted from other places. Better to have than not to have imo.